packages/usa-accordion/src/index.js

```diff
@@ -6,7 +6,8 @@ const { CLICK } = require("../../uswds-core/src/js/events");
 const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
 
 const ACCORDION = `.${PREFIX}-accordion, .${PREFIX}-accordion--bordered`;
-const BUTTON = `.${PREFIX}-accordion__button[aria-controls]`;
+const BANNER_BUTTON = `.${PREFIX}-banner__button`;
+const BUTTON = `.${PREFIX}-accordion__button[aria-controls]:not(${BANNER_BUTTON})`;
```

Comment:
All accordion buttons, except when combined with banner. This helps avoid conflicts because we now initialize banner.